contracts/crowdsale/ROUX_RefferalBasedCrowdsale.sol

```diff
@@ -24,9 +24,12 @@ contract ROUX_ReferralBasedCrowdsale is MintedCrowdsale, PostDeliveryCrowdsale,
 
     mapping(address => Backer) mBackers;
 
-    uint256 mMaxReferralPercentageFee = 3;
-    uint256 mMaxDiscount = 20;
-    uint256 oneHundred = 100;
+    // Maximum allowable referral reward percentage (i.e. 300 = 3%)
+    uint256 mMaxReferralPercentageFee = 300;
```

Comment:
Can we have a default instead of maximum?

Reply:
sounds good